packages/server/lib/modes/run.ts

```diff
@@ -335,6 +335,13 @@ async function compressRecording (options: { quiet: boolean, videoCompression: n
     return
   }
 
+  // if a user passes in videoCompression='true' into their config, coerce the value
+  // to the default CRF value which is 32
+  if (options.videoCompression === true) {
+    debug('coercing compression to 32 CRF')
+    options.videoCompression = 32
+  }
```

Comment:
I would expect mapping true to 32 to be handled configuration package. Is there a reason we can't accomplish this there vs overriding here?

By overriding this options, does it impact the configuration value reported at the end of the spec/run or is this only piping into the video compression? Does this alter the compression value logged to the terminal when `videoCompression= true`?

Reply:
> I would expect mapping true to 32 to be handled configuration package. Is there a reason we can't accomplish this there vs overriding here?

I'm not super familiar with us coercing values in the config package and not sure there is a place to do it. Any ideas?

> By overriding this options, does it impact the configuration value reported at the end of the spec/run or is this only piping into the video compression? 

It should only impact the option set inside [compressRecording](https://github.com/cypress-io/cypress/blob/develop/packages/server/lib/modes/run.ts#L691) since we create the object that is passed in. Since this is a boolean or a number and not an object, the ref change should happen by value and not reference.

>  Does this alter the compression value logged to the terminal when videoCompression= true?

if `videoCompression= true` it should log 'Compressing to 32 CRF' to the terminal